examples/destination/python/main.py

```diff
@@ -1,7 +1,10 @@
+import datetime
 from concurrent import futures
+
+from google.protobuf.timestamp_pb2 import Timestamp
 import grpc
+
 import read_csv
-import sys
```

Comment:
We deleted this sys import but using `sys.path.append('sdk_pb2')`

Reply:
fixed